share/completions/magento.fish

```diff
@@ -0,0 +1,720 @@
+__fish_magento_register_command_option setup:static-content:deploy -f -l no-images -d "Do not deploy images.";
+__fish_magento_register_command_option setup:static-content:deploy -f -l no-fonts -d "Do not deploy font files." 
+__fish_magento_register_command_option setup:static-content:deploy -f -l no-html -d "Do not deploy HTML files." 
+__fish_magento_register_command_option setup:static-content:deploy -f -l no-misc -d "Do not deploy other types of files (.md, .jbf, .csv, etc...).";
+__fish_magento_register_command_option setup:static-content:deploy -f -l no-html-minify -d "Do not minify HTML files.";
+__fish_magento_register_command_option setup:static-content:deploy -s t -l theme -d 'Generate static view files for only the specified themes. (default: ["all"]) (multiple values allowed)';
+__fish_magento_register_command_option setup:static-content:deploy -l exclude-theme -d 'Do not generate files for the specified themes. (default: ["none"]) (multiple values allowed)';
+__fish_magento_register_command_option setup:static-content:deploy -f -s l -l language -d 'Generate files only for the specified languages. (default: ["all"]) (multiple values allowed)';
+__fish_magento_register_command_option setup:static-content:deploy -f -l exclude-language -d 'Do not generate files for the specified languages. (default: ["none"]) (multiple values allowed)';
+__fish_magento_register_command_option setup:static-content:deploy -f -s a -l area -a "(__fish_print_magento_theme_areas)" -d 'Generate files only for the specified areas. (default: ["all"]) (multiple values allowed)';
+__fish_magento_register_command_option setup:static-content:deploy -f -l exclude-area -a "(__fish_print_magento_theme_areas)" -d 'Do not generate files for the specified areas. (default: ["none"]) (multiple values allowed)';
+__fish_magento_register_command_option setup:static-content:deploy -x -s j -l jobs -d "Enable parallel processing using the specified number of jobs. (default: 4)";
+__fish_magento_register_command_option setup:static-content:deploy -f -l symlink-locale -d "Create symlinks for the files of those locales, which are passed for deployment, but have no customizations";
+
+
+#
+# setup:store-config:set
+#
+__fish_magento_register_command_option setup:store-config:set -f -l base-url -a "(__fish_print_magento_url_protocols)" -d "URL the store is supposed to be available at";
+__fish_magento_register_command_option setup:store-config:set -f -l language -a "(__fish_print_magento_languages)" -d "Default language code";
+__fish_magento_register_command_option setup:store-config:set -f -l timezone -d "Default time zone code";
+__fish_magento_register_command_option setup:store-config:set -f -l currency -d "Default currency code";
+__fish_magento_register_command_option setup:store-config:set -f -l use-rewrites -d "Use rewrites";
+__fish_magento_register_command_option setup:store-config:set -f -l use-secure -d "Use secure URLs. Enable this option only if SSL is available.";
```

Comment:
What does it do if ssl isn't available? If it just won't do it then: 'Use secure URLs (if ssl is available)'.

Does anyone not have ssl available?